socorro/cron/jobs/matviews.py

```diff
@@ -90,48 +80,6 @@ class BuildADUCronApp(_MatViewBackfillBase):
     depends_on = ('fetch-adi-from-hive', 'reports-clean')
 
 
-class CrashesByUserCronApp(_MatViewBackfillBase):
-    proc_name = 'update_crashes_by_user'
-    app_name = 'crashes-by-user-matview'
```

Comment:
Are you sure we want to remove this? I'm not. 
Same applies to `CrashesByUserBuildCronApp`.

The reason is that we still haven't decided on a way to archive this. 
A couple of months ago we remove the UI for this and replaced it with the real-time one. 
Some day we need to export and archive up ALL of these historical records and from that day onwards, if someone wants to get historical data they can use the exported dump.